src/core/etl/tests/Flow/ETL/Tests/Benchmark/Row/Reference/Expression/AddJsonBench.php

```diff
@@ -0,0 +1,43 @@
+<?php declare(strict_types=1);
+
+namespace Flow\ETL\Tests\Benchmark\Row\Reference\Expression;
+
+use function Flow\ETL\DSL\lit;
+use function Flow\ETL\DSL\ref;
+use Flow\ETL\DSL\From;
+use Flow\ETL\DSL\To;
+use Flow\ETL\Flow;
+use Flow\ETL\Memory\ArrayMemory;
+use PhpBench\Attributes\Iterations;
+use PhpBench\Attributes\Revs;
+
+final class AddJsonBench
+{
+    #[Revs(100)]
+    #[Iterations(5)]
+    public function benchAddingJson() : void
```

Comment:
I think we should use snake case here, just like in PHPUnit tests. It should make the output more readable.